MEME/MEME/Model/MyPage/AskViewController.swift

```diff
@@ -75,74 +76,65 @@ class AskViewController: UIViewController {
+            make.height.equalTo(41)
+        }
         
-        view.addSubview(contentLabel)
         contentLabel.text = "문의내용"
-        contentLabel.font = UIFont(name: "Pretendard-Regular", size: 14)
-        NSLayoutConstraint.activate([
-            contentLabel.topAnchor.constraint(equalTo: emailLabel.bottomAnchor, constant: 26),
-            contentLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 24)
-        ])
-        
-        view.addSubview(askTextView)
-        askTextView.text = "   여기에 내용을 작성해주세요."
-        askTextView.textColor = UIColor.gray300
-        askTextView.font = UIFont(name: "Pretendard-Medium", size: 14)
+        contentLabel.font = .pretendard(to: .regular, size: 14)
+        contentLabel.textColor = .black
+        view.addSubview(contentLabel)
+        contentLabel.snp.makeConstraints { make in
+            make.top.equalTo(emailLabel.snp.bottom).offset(26)
+            make.leading.equalTo(view.snp.leading).offset(24)
+        }
+
+        askTextView.font = .pretendard(to: .medium, size: 14)
         askTextView.layer.cornerRadius = 9
-        
-        askTextView.delegate = self
-        
         askTextView.layer.borderColor = UIColor.gray300.cgColor
         askTextView.layer.borderWidth = 1
-        NSLayoutConstraint.activate([
-            askTextView.topAnchor.constraint(equalTo: contentLabel.bottomAnchor, constant: 10),
-            askTextView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 24),
-            askTextView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -24),
-            askTextView.heightAnchor.constraint(equalToConstant: 269)
-        ])
+        view.addSubview(askTextView)
+        askTextView.snp.makeConstraints { make in
+            make.top.equalTo(contentLabel.snp.bottom).offset(10)
+            make.leading.equalTo(view.snp.leading).offset(24)
+            make.trailing.equalTo(view.snp.trailing).offset(-24)
+            make.height.equalTo(269)
+        }
         
-        view.addSubview(sendButton)
         sendButton.setTitle("전송하기", for: .normal)
-        sendButton.backgroundColor = UIColor(named: "MainBold")
+        sendButton.backgroundColor = .mainBold
         sendButton.setTitleColor(UIColor.white, for: .normal)
         sendButton.layer.cornerRadius = 10
-        NSLayoutConstraint.activate([
-            sendButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -8),
-            sendButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            sendButton.heightAnchor.constraint(equalToConstant: 49),
-            sendButton.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.8)
-        ])
+        view.addSubview(sendButton)
+        sendButton.snp.makeConstraints { make in
+            make.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom).offset(-8)
+            make.centerX.equalTo(view.snp.centerX)
```

Comment:
```suggestion
            make.centerX.equalTo(view)
```
와 같이도 작성할 수 있으니 참고하시면 좋을 것 같습니다!